### DISCUSSION TITLE
✅ Nested Carousel just like the Loop Carousel

### DISCUSSION THREAD

**Discussion by ilkanellop** [elementor/elementor#22454] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I try to create a loop carousel with the current widget but i cannot achieve it because it lucks an option

### Describe the solution you'd like

It would be nice if it was possible to have custom loop carousel option and be able to create unique custom carousel that will explode the web design of our websites with this fetaure.

I suggest the following structure/options. In the layout at the choose template type add one more option with name Custom Loop and in the second option Choose a template add option Custom template. While this option is chosen the user will be able to add the number of the containers that he needs to create his custom carousel to build his own custom content inside the container carousel.

**Comment by nicholaszein** [comment]
Hello there !

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.14! 🥳 

You can now use the new [Nested Carousel](https://elementor.com/help/carousel-widget/) to create your own layouts.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
